sys/riscv/boot.c

```diff
@@ -59,23 +59,26 @@
  * will be dropped.
```

Comment:
Please update the description at the beginning of this file (DTB mapping is gone).

Reply:
Done!